### PR TITLE
DW-129 clean workspace when finished

### PR DESCRIPTION
**Pull request by cbernat**
Cleanup workspace after build complete.

### CODE REVIEW

**Comment by andresmoschini** [comment]
Is it so simple?

**Comment by cbernat** [comment]
> Is it so simple?

Seems to be 😄 only one way to know would be to test this.

**Comment by cbernat** [comment]
image
For this PR that is #616 in workspace the foder has been removed. Except temp one.

**Comment by andresmoschini** [comment]
>  For this PR that is #616 in workspace the foder has been removed. Except temp one.

Maybe temp folder is removed automatically in background.

I am pretty sure that @elsupergomez knows.

**Comment by cbernat** [comment]
> > For this PR that is #616 in workspace the foder has been removed. Except temp one.
> 
> Maybe temp folder is removed automatically in background.
> 
> I am pretty sure that @elsupergomez knows.

With the last fix it removes temp file also
image

File: Jenkinsfile [hunk 1]
```diff
@@ -88,6 +88,14 @@ pipeline {
             }
         }
     }
+    post { 
+        always {
```
Comment (elsupergomez) [inline Jenkinsfile]: Maybe is better do the clean in the *cleanup* and leave the *always* for other post actions

```suggestion
        cleanup{ 
```

**Comment by andresmoschini** [comment]
> I am pretty sure that @elsupergomez knows.

> Right now is the best solution, I will apply it to other projects.
> And we should follow this [JENKINS-41805](https://issues.jenkins-ci.org/browse/JENKINS-41805) to know if will handle by Jenkins in the future.

I knew! I knew!

**Pull request merged**